src/simple-cache-client.ts

```diff
@@ -700,8 +747,9 @@ export class SimpleCacheClient {
    * @param {CollectionTtl} [options.ttl] - How the TTL should be managed.
    * Refreshes the dictionary's TTL using the client's default if this is not
    * supplied.
-   * @returns {Promise<CacheDictionaryIncrement>} - The result of the increment
-   * operation (success or error).
+   * @returns {Promise<CacheDictionaryIncrement>} -
+   * {@link CacheDictionaryIncrement.Success} on success.
+   * {@link CacheDictionaryIncrement.Error} on failure.
```

Comment:
Important failure mode to document: the value to be incremented was not an integer or string representation of an integer